alpha/alpha.ino

```diff
@@ -8,95 +8,168 @@
 
 bool powerSaveTest = false;
 
+bool SavePower = false;
+int stackSize = 1000;
+unsigned long debounceDelay = 50; // the debounce time; increase if the output flickers
+
 void setup() {
   TS_HAL.begin();
   TS_HAL.ble_init();
-  
+
   // Reduce screen brightness to minimum visibility to reduce power consumption
   TS_HAL.lcd_brightness(12);
 
   TS_HAL.lcd_cursor(40, 0);
   TS_HAL.lcd_printf("ALPHA TEST");
 
-  if(powerSaveTest)
+  if (powerSaveTest)
   {
     TS_HAL.lcd_backlight(false);
     TS_HAL.lcd_sleep(true);
   }
 
   OT_ProtocolV2.begin();
-  
+
+  xTaskCreate(
+    traceTask, /* Task function. */
+    "Trace", /* name of task. */
+    stackSize * 10, /* Stack size of task */
+    NULL, /* parameter of the task */
+    2, /* priority of the task */
+    NULL); /* Task handle to keep track of created task */
+
+  xTaskCreate(
```

Comment:
might want to consider 'static task'

Reply:
since this is permanent

Reply:
I'll change this next time because I've never really used that before! Need to predefine the stack buffer too.